rasa/utils/tensorflow/model_data.py

```diff
@@ -160,6 +160,20 @@ def add_mask(self, key: Text, from_key: Text):
                 self.data[key].append(mask)
                 break
 
+    def add_lengths(self, key: Text, from_key: Text):
```

Comment:
please add return type and also a doc string

Reply:
Not sure, but I think the `add_mask` method is not used anymore. We might want to remove it.

Reply:
there isn't a return type because it doesn't return anything :) but I have added the docstring

Reply:
If there is no return value you should annotate it with `-> None`